src/CMakeLists.txt

```diff
@@ -363,6 +363,18 @@ ADD_SUBDIRECTORY(irc)
 
 ADD_DEFINITIONS(-DAPP_VERSION="${APP_VERSION}" -DAPP_BUILD="${APP_BUILD}" -DAPP_TARGET="${APP_TARGET}")
 
+IF(UNIX)
+    SET(DATAROOTDIR "${CMAKE_INSTALL_PREFIX}/share" CACHE STRING "The platform-specific directory in which read-only data is generally installed")
+    SET(DATADIR "${DATAROOTDIR}/mudlet" CACHE STRING "Directory in which mudlet installs its read-only data")
+    SET(LUA_DEFAULT_DIR "${DATADIR}/lua" CACHE STRING "Directory in which mudlet installs its read-only lua scripts")
+ENDIF(UNIX)
+
+# Define a preprocessor symbol with the default fallback location from which
+# to load installed mudlet lua files. Set LUA_DEFAULT_DIR to a
+# platform-specific value. If LUA_DEFAULT_DIR is unset, the root directory
+# will be used.
+ADD_DEFINITIONS(-DLUA_DEFAULT_PATH="${LUA_DEFAULT_DIR}")
```

Comment:
You have a fair point there.  A separate issue for that might be sensible after this goes in after all.

I know I did some experiments back when I was coding on my old PC, I'll have to take a look in the local repository contents there but I'm sure I had something though I don't recall if it went under INSTALL() :confused:  or not...  

*@vadi2* - this is currently targeting the `development` branch - do we also want it ported it to `release_30`?